tests/models/test_model.py

```diff
@@ -2560,7 +2562,7 @@ def test_hf_init(
         False,
     )
 
-    model = HuggingFaceModelWithFSDP(model, tokenizer)
+    model = HuggingFaceModel(model, tokenizer)
```

Comment:
would it be more correct to change this to BaseHuggingFaceModel (changing the args as appropriate)? glancing at what this was testing that might be better?